src/KlaviyoAPI.php

```diff
@@ -407,7 +407,7 @@ protected function createRequestBody( array $params )
      * @param array $params
      * @return array[]
      */
-    protected function createRequestJson( array $params)
+    protected function createRequestJson( $params)
```

Comment:
Nit: missing space at the end
```suggestion
    protected function createRequestJson( $params )
```